larray/core/group.py

```diff
@@ -1300,13 +1302,28 @@ def to_hdf(self, filepath, key=None, axis_key=None, **kwargs):
         --------
         >>> from larray import Axis
         >>> a = Axis("a=a0..a2")
-        >>> a.to_hdf('test.h5', 'a')
+        >>> a.to_hdf('test.h5')
         >>> a01 = a['a0,a1'] >> 'a01'
-        >>> a01.to_hdf('test.h5', 'a01', 'a')   # doctest: +SKIP
+
+        Save group
+
+        >>> # by default, the key is the name of the group
+        >>> # and axis_key the name of the associated axis
+        >>> a01.to_hdf('test.h5')                  # doctest: +SKIP
+
+        Save group with a specific key
+
+        >>> a01.to_hdf('test.h5', 'a_01')           # doctest: +SKIP
 
         Save group in a specific HDF group
 
-        >>> a.to_hdf('test.h5', 'groups/a', 'a')   # doctest: +SKIP
+        >>> a.to_hdf('test.h5', 'groups/a01')       # doctest: +SKIP
+
+        The associated axis is saved at the time with the group if not present in the HDF file yet
```

Comment:
I don't really understand this sentence. I guess you hesitated between "with the group" and "at the same time than the group" and ended up with a mix of both?

Reply:
Bad interactive rebase...